syz-cluster/dashboard/handler.go

```diff
@@ -167,6 +167,26 @@ func (h *dashboardHandler) patchContent(w http.ResponseWriter, r *http.Request)
 	h.streamBlob(w, patch.BodyURI)
 }
 
+func (h *dashboardHandler) findingInfo(w http.ResponseWriter, r *http.Request) {
+	finding, err := h.findingRepo.GetByID(r.Context(), r.PathValue("id"))
+	if err != nil {
+		http.Error(w, fmt.Sprint(err), http.StatusInternalServerError)
```

Comment:
fmt.Sprint(err) -> err.Error() plz

Reply:
Refactored `handler.go` and pushed it as a separate commit.